### PR TITLE
Lagos-2020

### PR DESCRIPTION
**Pull request by haroldsphinx**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

DevOpsDays Lagos, Nigeria is going to be the first Official DevOpsDays Conference Happening in Nigeria, the organizers of this conference have successfully held local devops meetups and conference and would love to take this up a notch by organizing an internation DevOpsDays Conference in Nigeria

### CODE REVIEW

File: data/events/2020-lagos.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    linkedin: "https://www.linkedin.com/in/sallyfields"
+    website: "https://mattstratton.com"
+    image: "sally-fields.jpg"
+
+organizer_email: "lagos@devopsdays.org" # Put your organizer email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  - id: samplesponsorname
```
Comment (phrawzty) [inline data/events/2020-lagos.yml]: You'll want to comment the sample sponsor out.

File: data/events/2020-lagos.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    level: gold
+ #  url: http://mysponsor.com/?campaign=me # Use this if you need to over-ride a sponsor URL.
+  - id: arresteddevops
+    level: community
+
+sponsors_accepted : "yes" # Whether you want "Become a XXX Sponsor!" link
```
Comment (phrawzty) [inline data/events/2020-lagos.yml]: You should set this to `no` until you're ready to accept sponsors (i.e. you have a prospectus prepared).

File: data/events/2020-lagos.yml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
```
Comment (phrawzty) [inline data/events/2020-lagos.yml]: You should comment this until you're ready to accept sponsors (i.e. you have a prospectus prepared).

File: data/events/2020-lagos.yml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+registration_date_start: # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```
Comment (phrawzty) [inline data/events/2020-lagos.yml]:
```suggestion
coordinates: "6.45407, 3.39467" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
```

File: data/events/2020-lagos.yml [hunk 5]
```diff
@@ -0,0 +1,90 @@
+name: "2020-lagos" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2020" # The year of the event. Make sure it is in quotes.
+city: "Lagos" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdayslagos" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
```
Comment (phrawzty) [inline data/events/2020-lagos.yml]:
```suggestion
# event_twitter: "devopsdayslagos" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
```

Fill this out later, when the account actually exists.

image

File: content/events/2020-lagos/welcome.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+<!-- <div class = "row">
+  <div class = "col-md-2">
+    <strong>Speakers</strong>
+  </div>
+  <div class = "col-md-8">
+    Check out the {{< event_link page="speakers" text="speakers!" >}}
+  </div>
+</div> -->
+
+<div class = "row">
```
Comment (phrawzty) [inline content/events/2020-lagos/welcome.md]: Comment this out until you're ready to accept sponsors (i.e. have a prospectus ready).

File: content/events/2020-lagos-nigeria/welcome.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+<!-- <div class = "row">
+  <div class = "col-md-2">
+    <strong>Speakers</strong>
+  </div>
+  <div class = "col-md-8">
+    Check out the {{< event_link page="speakers" text="speakers!" >}}
+  </div>
+</div> -->
+
+<div class = "row">
```
Comment (phrawzty) [inline content/events/2020-lagos-nigeria/welcome.md]: Comment this until you're ready to accept sponsors (i.e. have a prospectus prepared).

File: data/events/2020-lagos-nigeria.yml [hunk 8]
```diff
@@ -0,0 +1,90 @@
+name: "2020-lagos-nigeria" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
```
Comment (phrawzty) [inline data/events/2020-lagos-nigeria.yml]: This appears to be a duplicate of `2020-lagos.yml`. Please remove it (unless it's being used for something I don't understand—in which case, please explain).

File: data/events/2020-lagos.yml [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+    image: "adedayo.jpg"
+  - name: "Ewere Diagboya"
+    twitter: "@nimboya"
+    employer: "Seamfix"
+    github: "nimboya"
+    image: "ewere.jpg"
+  - name: "Idowu Emehinola"
+    employer: "Deimos Technology"
+    twitter: "@_hydeenoble"
+    github: "devopsdays"
```
Comment (adrianmoisey) [inline data/events/2020-lagos.yml]: This needs to be removed, or changed to Idowu's actual GitHub profile

**Pull request merged**